### DISCUSSION TITLE
How can I use an external search box instead of the default one?

### DISCUSSION THREAD

**Discussion by Restingo** [rappasoft/laravel-livewire-tables#315] [Q&A] [answered]
With DataTables for example I can use dom and use external input box for searching. Is this possible with this package?

**Comment by rappasoft** [comment, marked as answer]
You can disable the search box using `$showSearch` and include `filters[search]` in the URL from anywhere else. Or you can create your own search box and `wire:model` it to `filters.search`.

**Reply by Restingo** [reply]
Thanks, that worked really well!
